### DISCUSSION TITLE
All the music in uncompressed WAV

### DISCUSSION THREAD

**Discussion by rlk** [Neverball/neverball#259] [Show and tell]
Recently I pulled the old Roland Fantom S out of storage and had it render out all the content on the memory card from 2003. I added them to the repo [HERE](https://github.com/Neverball/neverball-music/tree/master/wav). They're uncompressed 16-bit 44.1KHz stereo samples, so you can use them, remix them, or reencode them as you like.

That includes the 5 level backgrounds and the 2 pieces of incidental music, plus an extra 6 background and incidental bits that I composed around the same time but didn't include. 18 years ago? Dang.

**Comment by parasti** [comment]
Congrats, by the way! I'm a little bummed that I couldn't do a release to celebrate, but work/life have a mind of their own.

**Comment by qwertychouskie** [comment]
I noticed that Track 5 is a completely different tempo compared to in-game.  Was this intentional or an export bug?

**Reply by rlk** [reply]
Oh it is! Interesting, that's a completely different mix of that track.

I'll dig around to see if I still have the original.

**Comment by KodeBreaker-Has-SWAG** [comment]
I really like Extra 1 and Extra 3. They sound really nice!

**Comment by Adacrop-The-Great** [comment]
Those are excellent tracks! 👍 You should also join the NeverCord: [https://discord.gg/G6Cud6ytr8](url)
